web/src/click_handlers.js

```diff
@@ -763,8 +763,10 @@ export function initialize() {
                 return;
             }
 
-            e.preventDefault();
-            e.stopPropagation();
+            if (!pm_list.is_private_messages_collapsed()) {
+                e.preventDefault();
+                e.stopPropagation();
+            }
             const $left_sidebar_scrollbar = $(
                 "#left_sidebar_scroll_container .simplebar-content-wrapper",
```

Comment:
```js
    $("body").on(
        "click",
        ".private_messages_container.zoom-out #private_messages_section_header",
        (e) => {
            if ($(e.target).closest("#show_all_private_messages").length > 0) {
                // Let the browser handle the "all direct messages" widget.
                return;
            }

            e.preventDefault();
            e.stopPropagation();
```

Let's change this to above.

Reply:
Thanks a lot! Yeah, this is actually quite useful in this case and works as expected.